packages/process/src/node/process.ts

```diff
@@ -125,6 +127,20 @@ export abstract class Process {
         return this._killed;
     }
 
+    get exited(): boolean {
```

Comment:
Since these `get`ters and `set`ters just pass through to the protected value, `exited` and `attached` could just be public fields.

Reply:
Updated.